__tests__/app.test.js

```diff
@@ -322,3 +323,78 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/reviews (queries)", () => {
+  test("200: returns an array of review objects where the order defaults to descending and the sort by category is created_at", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("200: returns an array of review objects a query of a specific category is searched for", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toHaveLength(1);
+        expect(reviews[0].category).toBe("dexterity");
+      });
+  });
+  test("200: returns an array of review objects which is sorted by the specified sort_by column in the query", () => {
+    return request(app)
+      .get("/api/reviews/?sort_by=votes")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSortedBy("votes", { descending: true });
+      });
+  });
+  test("200: returns an array of review objects which is ordered in the specified way as per the query", () => {
+    return request(app)
+      .get("/api/reviews/?order=asc")
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSortedBy("created_at", { ascending: true });
+      });
+  });
+  test("200: sort_by and order queries work in conjunction", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=review_id&order=desc")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSortedBy("review_id", { descending: true });
+      });
+  });
```

Comment:
Please write another test where the category is valid, but has no reviews. For example: `?category=children's games` This should expect a status 200 and an empty array back.